src/tasks/start/frontend/generate-artifacts.ts

```diff
@@ -20,17 +20,10 @@ export async function generateAppArtifacts(
   artifacts: TruffleEnvironmentArtifacts
 ): Promise<void> {
   await _copyManifest(appBuildOutputPath)
-  await _generateUriArtifacts(appBuildOutputPath, artifacts)
+  await generateUriArtifacts(appBuildOutputPath, artifacts)
```

Comment:
Exposing artifact generation so that it can be called when the backend watcher recompiles smart contracts.